WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -80,20 +80,25 @@ public WalletViewModel(bool receiveDominant)
 			advancedAction.Items.Add(infoTab);
 			advancedAction.Items.Add(buildTab);
 
-			// Open and select tabs.
+			// Open tabs.
 			sendTab?.DisplayActionTab();
-			if (receiveDominant)
+			receiveTab.DisplayActionTab();
+			coinjoinTab.DisplayActionTab();
+			historyTab.DisplayActionTab();
+
+			// Select tab
+			if (receiveDominant || global.UiConfig.Tab == receiveTab.Title)
```

Comment:
I think that we can probably use CurrentTab.GetType().ToString() and save that.

Then to restore, `var tabType = Type.GetType(typestring from previous step)`

Then open the tab like shell.OpenDocument(tabType) should be possible I beleive.

Reply:
Otherwise everytime we add a new type of tab we are going to have to modify this code to do a string check, and someone will forget and this will break.

Reply:
I don't agree because the Type is one and there could be more than one tab of the same type in the future.

Reply:
`Otherwise everytime we add a new type of tab we are going to have to modify this code to do a string check, and someone will forget and this will break.`

@danwalmsley 
This PR makes Wasabi display the last active tab if it is one of the following tabs: Send, Receive, CoinJoin, History.
If the last active tab is not one of the mentioned tabs then Wasabi will display the CoinJoin tab.

The change proposed will be affected only if we remove one of those tabs, but if we add new tabs it will work just fine.

Reply:
> I don't agree because the Type is one and there could be more than one tab of the same type in the future.

I dont think that is true,

In future, we might have a settings file that says:

OpenWallets
{
      {
            FilePath: MyWallet1.json,
            OpenTabs: TabType1, TabType2, TabType3
       }
}

If we want to move away from using Type to reference which DocumentTab to open, then we need to move to something like having a unique ID to identify them, but right now this doesnt seem to be necessary?